test/chart/pie_chart/pie_chart_data_test.dart

```diff
@@ -73,7 +73,7 @@ void main() {
                 PieChartSectionData(value: 22, color: Colors.green),
               ],
             ),
-        false,
+        true,
```

Comment:
This case expected the `PieChartSectionData`s to be unequal, even though their `sections` contain exactly the same elements in the same order. Technically this could be considered a breaking change if we merge it.

Reply:
I saw that in the Flutter Framework GitHub any change that breaks a test is considered a "breaking change". 

It might break functionality that expect `section` to be unequal when they don't have the same reference. 

I think it's very unlikely that this will actually happen for anyone. But communicating this as a breaking change ensures trust in consumers of `fl_chat`.

Reply:
Makes sense. Thanks for mentioning that. 
So let's consider it as a breaking change.